app/views/book/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="col-md-offset-2 col-md-8">
   <div class="book-header <%= current_user? ? '' : 'logged' %>">
-    <h1>ム mumuki</h1>
+    <h1>ム mumuki <%= Organization.current.title_suffix %></h1>
```

Comment:
Just nitpicking, but this would end up with 2 spaces between mumuki and the suffix's dash. We don't want that, right?

Reply:
Totally. Fixed now!